README.md

```diff
@@ -1,2 +1,186 @@
-# Fetching-Bot-Rooh-
-Srishti 2020
+# **Fetching-bot-Rooh**
+---
+
+## Abstract
+
+The main objective of our bot is to go and fetch any particular object in a defined area. It works in the following manner:
+*   Search for the QR code on the object in defined area
+*    Scans the QR code just for verifying the object
+*    Calculate the distance between it and the object
+*  Go and pick the object
+*   And at last, returns to its starting position
+
+
+![Final Product](https://github.com/bhavesh0902/Fetching-Bot-Rooh-/blob/master/Images%20%26%20Videos/Images/Picture1.jpg)
+
+---
+
+## Motivation
+It can be used for general household purposes and even in distribution chain of any product, thus making our job easy.
+
+## Components
+
+* 2 NORMAL WHEELS 
+* 1 CASTOR WHEEL 
+* ULTRASONIC SENSOR 
+* ARDUINO UNO  
+* RASPBERRY PI 3  
+* CRYTON MOTOR DRIVER 
+* 2 JOHNSON MOTOR      
+* 2 SERVO MOTOR    
+* SD CARD 
+
+---
```

Comment:
Flowchart is missing here. Add flowchart under heading "Workflow" after "Motivation"